astropy/coordinates/tests/test_pickle.py

```diff
@@ -69,3 +70,27 @@ def test_simple_object(pickle_protocol, name, args, kwargs, xfail):
         pytest.xfail()
     original = name(*args, **kwargs)
     check_pickling_recovery(original, pickle_protocol)
+
+
+class _CustomICRS(coord.ICRS):
+    default_representation = coord.PhysicsSphericalRepresentation
+
+
+@pytest.mark.parametrize(
+    "frame",
+    [
+        coord.SkyOffsetFrame(origin=coord.ICRS(0*u.deg, 0*u.deg)),
+        coord.SkyOffsetFrame(5*u.deg, 10*u.deg, origin=coord.Galactic(2*u.deg, -3*u.deg)),
+        coord.SkyOffsetFrame(5*u.deg, 10*u.deg, 10*u.pc,
+                             origin=coord.Galactic(2*u.deg, -3*u.deg),
+                             representation_type=coord.PhysicsSphericalRepresentation),
+        coord.SkyOffsetFrame(5*u.deg, 10*u.deg, 0*u.pc,
```

Comment:
This adds our test classes to the frame transform graph. Do we care? Should we clean up? (I.e., should we have a `teardown_module` function?)